frontend/src/hooks/usePagination.ts

```diff
@@ -20,7 +20,7 @@ const usePagination = <T>(
         }
 
         const result = paginate(dataToPaginate, limit);
-        setPageIndex(0);
+        // setPageIndex(0);
```

Comment:
This looks very subtle, what else does this effect?

Reply:
hopefully nothing. I could not find any other usage of this hook. It solved all my pagination needs so I decided to use it.

Reply:
Okay deprecated hook. Doesn't seem to be in use anywhere